package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "6.10.0",
+  "version": "8.0.0",
```

Comment:
What happened to v7?

Reply:
The version of the main branch is `7.0.37` https://github.com/bcgov/business-filings-ui/blob/d455e431ee1224de8a298aaa9bb8e251ee52162a/package.json#L3

I believe a major upgrade was made when the business digital card was added.

Reply:
Ah, sorry for all the dumb questions. This is perfect. Thanks.